pkg/lib/authn/otp/sender.go

```diff
@@ -41,7 +41,7 @@ type TranslationService interface {
 
 type Sender interface {
 	SendEmailInNewGoroutine(ctx context.Context, msgType translation.MessageType, opts *mail.SendOptions) error
-	SendSMSInNewGoroutine(ctx context.Context, msgType translation.MessageType, opts *sms.SendOptions) error
+	SendSMSImmediately(ctx context.Context, msgType translation.MessageType, opts *sms.SendOptions) error
```

Comment:
Changing this will have an great impact on the duration of database transaction.

I think we need to first restructure authflow code to make it support running I/O operation without the context of a ongoing database transaction.

Reply:
Per offline discussion, we will change the method signature of ReactTo to support delayed one-time function.

@tung2744

1. Keep SendSMSInNewGoroutine (for latte) in 
2. Introduce SendSMSImmediately
3. Split `Send` in pkg/lib/authn/otp/sender.go into `SendImmediately` and `SendInNewGoroutine`.
4. authflow uses SendImmediately, while latte uses SendInNewGoroutine